app/models/purchase.py

```diff
@@ -57,12 +57,28 @@ def __init__(self, cart_items):
     def __repr__(self):
         return "<Order: {}>".format(self.id)
 
+    def get_all_purchases(self):
```

Comment:
Why do we need a `get_all_purchases` function? `{order}.purchases` should work the same